src/camtasia/project.py

```diff
@@ -56,7 +56,12 @@ def timeline(self) -> Timeline:
     @property
     def _project_file(self):
         "The project's main JSON data file, i.e. the 'tscproj' file."
-        return self.file_path / 'project.tscproj'
+        if self.file_path.is_dir():
+            for file in self.file_path.iterdir():
+                if file.is_file() and file.suffix == '.tscproj':
+                    return file
+        else:
+            return self.file_path
```

Comment:
This needs to account for the possibility that a) file_path is a directory and b) no matching .tscproj file is found in it. As currently coded, this function will return `None`, which will lead to a confusing error message. It's probably sufficient to just raise a `FileNotFoundError`.